src/VisualStudio/Core/Def/Implementation/ProjectSystem/Legacy/AbstractLegacyProject_IIntellisenseBuildTarget.cs

```diff
@@ -2,19 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Immutable;
-using System.IO;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis.Internal.Log;
 using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.Interop;
-using Roslyn.Utilities;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.Legacy
 {
     internal partial class AbstractLegacyProject : IIntellisenseBuildTarget
     {
+        void IIntellisenseBuildTarget.SetIntellisenseBuildResult(bool succeeded, string reason)
+        {
+        }
+
+#if false
```

Comment:
que??

Reply:
Code was already commented out in weird fashion. I did not want to delete the code as I wasn't sure if it was intended to be kept for reference, so just `#if false` the entire commented code.